src/ruby/yast/logger.rb

```diff
@@ -1,6 +1,10 @@
 require "yastx"
 require "yast/builtins"
 
+# FIXME: a workaround for https://github.com/yast/yast-ruby-bindings/issues/133
+require "enc/encdb.so"
+require "enc/trans/transdb.so"
```

Comment:
Um, I played with it a bit but I did not succeed... :worried: 

I tried different places for the `require` calls, also tried using `Init_enc()` [as mentioned here](http://stackoverflow.com/questions/25449061/how-do-you-fully-initialize-an-embedded-ruby-vm-in-a-c-application), tried several places, but with the same result - the exception was still raised.

Only when placing the `require`s in the Ruby file worked.

I do not want to spend more time here, let's use this workaround and improve it later when really needed....